renderer/main.js

```diff
@@ -1213,11 +1213,7 @@ function openTorrentContextMenu (infoHash) {
 }
 
 function getTorrentPath (torrentSummary) {
-  var itemPath = path.join(torrentSummary.path, torrentSummary.files[0].path)
-  if (torrentSummary.files.length > 1) {
-    itemPath = path.dirname(itemPath)
-  }
-  return itemPath
+  return path.join(torrentSummary.path, torrentSummary.name)
```

Comment:
That's pretty bad. Just tried dragging two files in, I noticed that the file paths was like `Downloads/file1`, and the torrent path was `/Users/mathias`. It would make more sense to make the torrent path `/Users/mathias/Downloads` and the file paths simply `file1`. Does this make sense?

This way I think it would be easier to avoid deleting the dowloads folder.